test/defaultHttpClientTests.ts

```diff
@@ -163,7 +163,7 @@ describe("defaultHttpClient", function () {
       download.notified.should.be.true;
     });
 
-    it("for blob or stream bodies", async function () {
+    it.skip("for blob or stream bodies", async function () {
```

Comment:
reason to skip this test?

Reply:
Reporting downloading/uploading is kind of hard with Fetch right now. The Storage Explorer team doesn't need it so I didn't spend too much time right now on it.

Reply:
If Fetch solves their problems I'll try to finish all the functionality.

Reply:
So here we are going backwards. I think this may affect storage js sdk. We will break them.

Reply:
I'm slightly surprised they don't require progress reporting..don't they have progress bars for uploads and downloads?

Reply:
On the node side I thought this was already done by wrapping the body streams to provide reporting. As long as fetch lets you provide and receive a node-style stream for the request and response body it should be doable. And on the browser side it seems like you'll continue using XhrHttpClient(?) so that's not a concern.